cmd/kubelet/app/options/options.go

```diff
@@ -29,6 +30,26 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	defaultRootDir             = "/var/lib/kubelet"
+	experimentalFlannelOverlay = false
+
+	// When these values are updated, also update test/e2e/framework/util.go
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "3.0"
+	// Auto detect cloud provider.
+	AutoDetectCloudProvider         = "auto-detect"
+	DefaultKubeletPodsDirName       = "pods"
+	DefaultKubeletVolumesDirName    = "volumes"
+	DefaultKubeletPluginsDirName    = "plugins"
+	DefaultKubeletContainersDirName = "containers"
```

Comment:
cc @mikedanese A bunch of kubelet const variables were moved to `pkg/apis/componentconfig/v1alpha1/defaults.go`. Can you help us understand the reasoning, and guidance for where to put consts like `DefaultKubeletVolumesDirName`?